resources/views/sensor/show.blade.php

```diff
@@ -0,0 +1,60 @@
+@extends('layouts.dashboard')
+
+@section('title', 'Show Sensor')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12 col-sm-12 col-md-10 col-lg-10 col-xs-offset-0 col-sm-offset-0 col-md-offset-1 col-lg-offset-1">
+            <div class="panel panel-primary">
+                <div class="panel-heading">
+                    <h3 class="panel-title">Sensor</h3>
+                </div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-md-12 col-lg-12 hidden-xs hidden-sm">
+                            <strong>{{ $sensor->name }}</strong><br>
+                            <table class="table table-sensor">
+                                <tbody>
+                                <tr>
+                                    <td>ID:</td>
+                                    <td>{{ $sensor->id }}</td>
+                                </tr>
+                                <tr>
+                                    <td>Device:</td>
+                                    <td><a href="{{ route('device.show', $sensor->device_id) }}">{{ $sensor->device->name ?? 'null' }}</a></td>
+                                </tr>
+                                <tr>
+                                    <td>Type:</td>
+                                    <td>{{ $sensor->type }}</td>
+                                </tr>
+                                <tr>
+                                    <td>Latest Value:</td>
+                                    <td><a href="{{ route('sensor.show', $sensor->latest_data->id ?? '0') }}">{{ $sensor->latest_data->value ?? 'null' }}</a></td>
```

Comment:
Clicking the latest value should redirect you to the sensordata show not the sensor show.

Reply:
fixed in 7d32ed6